.github/workflows/build_test_and_release_app.yml

```diff
@@ -63,5 +63,5 @@ jobs:
         run: npm ci
       - name: Release and tag a new version
         env:
-          SLACK_WEBHOOK_URL: ${{ github.ref == 'refs/heads/main' && secrets.SLACK_WEBHOOK_URL || github.ref == 'refs/heads/dev' && secrets.SLACK_WEBHOOK_URL_BETA }}
+          SLACK_WEBHOOK_URL: ${{ github.ref == 'refs/heads/main' && secrets.SLACK_WEBHOOK_URL }}
```

Comment:
Why &&?

Reply:
If we have now only one channel of versions for Slack, I can leave only the needed hook (since we gave up on the `dev` branch idea).

Reply:
removed